source/linux/Foundational_Components/Power_Management/pm_dfs.rst

```diff
@@ -332,6 +332,7 @@ table.
                                 };
 
         To disable any of the above OPP's one can simply add
+
```

Comment:
The indentation of code-blocks directive doesn’t follow - https://github.com/TexasInstruments/processor-sdk-doc/blob/master/CONTRIBUTING.md#indentation-and-whitespace